zebra-network/src/peer/connection.rs

```diff
@@ -884,6 +899,9 @@ where
     #[instrument(name = "msg_as_req", skip(self, msg), fields(%msg))]
     async fn handle_message_as_request(&mut self, msg: Message) {
         trace!(?msg);
+
```

Comment:
```suggestion
```

Is the `trace!` and the `debug!` needed ? If so, maybe remove the space between them ?

Reply:
Yes, `%msg` is a shorter summary of the message, `?msg` is the full message data.